app/styles/components/c-support-form.scss

```diff
@@ -0,0 +1,140 @@
+@import '../settings';
+
+.c-support-form {
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+
+  h1 {
+    color: $color-4;
+    font-size: $font-size-x-normal;
+    font-weight: $font-weight-bold;
+    line-height: 2.21;
+    margin-bottom: 20px;
+    margin-top: 0;
+    text-align: center;
+  }
+
+  form {
+
+    div {
+      float: left;
```

Comment:
Why are you using this? Only use `float` when you are very sure you need it, ok? Otherwise this can give you more problems than solutions.